src/OpenTK/Platform/X11/API.cs

```diff
@@ -52,6 +52,13 @@ namespace OpenTK.Platform.X11
 
     #endregion
 
+    // X11 has some defined values
+    internal static class Consts
+    {
+        public static readonly IntPtr None = IntPtr.Zero;
+        public static readonly IntPtr CurrentTime = IntPtr.Zero;
+    }
+
```

Comment:
What do these mean? It has some consts, but you've defined them as zero. Why can't we use `IntPtr.Zero` instead of None? Why is `CurrentTime` an `IntPtr` that's zero?